contracts/chain-adapters/l2/Arbitrum_WithdrawalHelper.sol

```diff
@@ -23,8 +23,8 @@ contract Arbitrum_WithdrawalHelper is WithdrawalHelperBase {
     // Error which triggers when the supplied L1 token does not match the Arbitrum gateway router's expected L2 token.
     error InvalidTokenMapping();
 
-    modifier onlyFromHubPool() {
-        require(msg.sender == CrossDomainAddressUtils.applyL1ToL2Alias(HUB_POOL), "ONLY_HUB_POOL");
+    modifier onlyCrossDomainAdmin() {
+        require(msg.sender == CrossDomainAddressUtils.applyL1ToL2Alias(crossDomainAdmin), "ONLY_CROSS_DOMAIN_ADMIN");
```

Comment:
Should this just be relocated down to the `_requireAdminSender()` function below, rather than having it invoke this as a modifier?

Reply:
Yeah that's a good call. I took it out of the spoke pool, but it just seems like extra diff that we can cut out. Done here: https://github.com/across-protocol/contracts/pull/605/commits/167b8a9ddeff09555bb627b7a6419c1cffea36d5